src/Tizen.NUI/src/public/Theme/ThemeManager.cs

```diff
@@ -324,7 +332,7 @@ internal static void ApplyExternalPlatformTheme(string id, string version)
             }
 
             // If no platform theme has been applied and the base theme can cover the given one, do nothing.
-            if (platformTheme == null && baseTheme.HasSameIdAndVersion(id, version))
+            if (platformTheme == null && baseTheme != null && baseTheme.HasSameIdAndVersion(id, version))
```

Comment:
baseTheme can be nullable?

Reply:
It was not in the previous code. But to reduce memory usage (for VD), I decided to make it nullable. I also edited all lines referencing `baseTheme` to avoid null exception.